cmd/run.go

```diff
@@ -58,9 +62,12 @@ func init() {
 	runCmd.Flags().StringVarP(&sshKeyPath, "ssh-key-path", "i", "", "path to private ssh key")
 	runCmd.Flags().UintVarP(&sshTimeout, "ssh-connect-timeout", "c", 30, "time to wait (in secs) for hosts to accept connection")
 	runCmd.Flags().UintVarP(&timeout, "timeout", "t", 300, "total time to wait (in secs) for host processing to complete")
+	runCmd.Flags().StringSliceVarP(&certIPs, "cert-ips", "", []string{}, "list of ips in ca cert (comma-delimited)")
+	runCmd.Flags().StringSliceVarP(&certDNSNames, "cert-dns-names", "", []string{}, "list of dns names in ca cert (comma-delimited)")
```

Comment:
Are we going to add more than one DNS Name? 

Inside _pkg/x509/certificate.go_, this value is used to set `certTmpl.DNSNames` and the first element is used to set of the `CommonName` of the cert subject.

- Does this complect two separate fields?
- What if the Nth element, and not the 1st, is the intended `CommonName`?

If we only need the 1 element then we can ingest a single string here and convert it into a single element slice inside the func `CreateSelfSignedCert`.

Reply:
This will allow us to support multiple SANs in the cert, which could be useful. We need to support at least a single SAN anyhow due to some oddities in golang's ssl validation that can cause the lone CN not to be validated properly in certain circumstances, so no reason not to use the slice and get full SAN support for free.

I did add a comment in the arg help to make it clear that the first one is also the CN.